helpers/prerequisites.psm1

```diff
@@ -119,7 +119,8 @@ Function Test-NetStackPrerequisites {
                     else { ($TargetInfo | Where-Object Name -eq $thisTarget).Module = $false }
 
                     if ($EnableFirewallRules) {
-                        $ModuleBase = (Get-Module Test-NetStack -CimSession $thisTarget -ListAvailable | Select-Object -First 1).ModuleBase
+                        $psSession = New-PSSession -ComputerName $thisTarget

```

Comment:
No issue making the change but unclear what is gained by it. Do you cleanup the pssession later?

Reply:
added a line to clean up!